xc/xenops_server_xen.ml

```diff
@@ -728,7 +728,7 @@ module VM = struct
 		| Reboot -> Domain.Reboot
 		| PowerOff -> Domain.PowerOff
 		| Suspend -> Domain.Suspend
-		| Halt -> Domain.Halt
+		| Halt -> Domain.PowerOff
```

Comment:
@robhoes would there be a problem with the operation afterwards where it expects the domain to need a poweroff?: https://github.com/xapi-project/xenopsd/blob/master/lib/xenops_server.ml#L847